PRFCS/prfc-2.md

```diff
@@ -131,40 +135,39 @@ Log `Transfer` must be triggered if `transfer` is successful.
 ```rust
 fn transfer_from(from_address: PublicAddress, to_address: Option<PublicAddress>, token_id: TokenID)
 ```
-
-Transfers the token identified by `token_id` from the account identified by `from_address`, to the account identified by `to_address`, on behalf of the token owner. If `to_address` is None, the token will be burnt.
+Transfers the token identified by `token_id` to the account identified by `to_address` on behalf of the token owner identified by `from_address`. If `to_address` is None, the token will be burnt.
 
 `transfer_from` must panic if: 
-1. Some(`txn.signer`) != get_spender(token_id).
-2. `from_address` != get_owner(token_id).
+1. Some(`calling_account`) != `spender(token_id)`.
+2. `from_address` != `owner(token_id)`.
 3. Or, if evaluating (1.) or (2.) causes a panic.
 
 Log `Transfer` must be triggered if `transfer_from` is successful. 
 
 ### set_spender
 
 ```rust
-fn set_spender(token_id: TokenID, spender_address: PublicAddress)
+fn set_spender(token_id: TokenID, spender: PublicAddress)
 ```
 
-Gives the account identified by `spender_address` the right to transfer the token identified by `token_id` on behalf of its owner.
+Grants the account identified by `spender` the right to transfer the token identified by `token_id` on behalf of its owner.
 
 `set_spender` must panic if:
-1. `txn.signer` != get_owner(token_id).
+1. `calling_account` != `owner(token_id)`.
 2. Or, if evaluating (1.) causes a panic.
 
 Log `SetSpender` must be triggered if `set_spender` is successful.
 
 ### set_exclusive_spender
 
 ```rust
-fn set_exclusive_spender(spender_address: PublicAddress)
+fn set_exclusive_spender(spender: PublicAddress)
 ```
 
-Gives the account identified by `spender_address` the right to transfer *all* tokens owned by `txn.signer`. Calling this method MUST have the same effects as calling `set_spender` for every token owned by `txn.signer` with the same `spender_address`.
+Gives the account identified by `spender` the right to transfer *all* tokens owned by `calling_account`. Calling this method MUST have the same effects as calling `set_spender` for every token owned by `calling_account` with the same `spender`.
```

Comment:
**Grants** the account identified by `spender` the right to transfer *all* tokens owned by **the** `calling_account`.